vignettes/advanced.Rmd

```diff
@@ -268,15 +466,161 @@ pool(anl_obj)
 
 # Delta Adjustment
 
+A delta-adjustment refers to a shift in the distribution of the (unobserved) outcome values by a certain amount. This might be used to perform sensitivity analyses.
```

Comment:
```suggestion
A delta-adjustment refers to a shift in the distribution of the (imputed) outcome values by a certain amount. This can be used to perform sensitivity analyses.
```